libs/plugin-scanner/build.gradle

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// This is only required because :server needs this at runtime.
+// We'll be removing this in 8.0 so for now just publish the JAR to make dependency resolution work.
```

Comment:
is this comment actually true? I think this is outdated copied from somewhere?

Reply:
I think we always need to publish this

Reply:
that was a copy paste. good spot